src/modules/components/proposal/proposalActions/proposalActionsList/proposalActionChangeMembers/proposalActionChangeMembers.test.tsx

```diff
@@ -58,4 +61,15 @@ describe('<ProposalActionChangeMembers /> component', () => {
         render(createTestComponent());
         expect(screen.getByText(modulesCopy.proposalActionChangeMembers.blockNote)).toBeInTheDocument();
     });
+
+    it('renders the correct block explorer link for removed memeber', () => {
+        const type = ProposalActionType.REMOVE_MEMBERS;
+        const currentMembers = 7;
+        const members = [{ address: '0x742d35Cc6634C0532925a3b844Bc454e4438f44e' }];
+        const action = generateProposalActionChangeMembers({ type, currentMembers, members });
```

Comment:
Let's remove vars that have no effect for the test
```suggestion
        const members = [{ address: '0x742d35Cc6634C0532925a3b844Bc454e4438f44e' }];
        const action = generateProposalActionChangeMembers({ members });
```